src/brevitas/core/stats/stats_op.py

```diff
@@ -24,21 +24,28 @@
 
 DEFAULT_STD_DEV_EPSILON = 1e-8
 
+# When restricting scales to powers of two, signed statistics cannot be used because
```

Comment:
I understand this comment but it is too specific.
The point of this is to simply keep track of which scales have a sign or not, all the logarithmic check part is someone else responsibility so I would avoid making this comment too long/complex